tests/end2end-tests/graphstorm-lp/mgpu_test.sh

```diff
@@ -1052,3 +1052,21 @@ error_and_exit $?
 rm -fr /data/gsgnn_lp_ml_transe_l2/*
 
 rm -fr /tmp/*
+
+echo "**************dataset: MovieLens: LP, RGCN layer: 1, node feat: fixed HF BERT, BERT nodes: movie, edge feat: user,rating,movie:feat inference: mini-batch"
+python3 -m graphstorm.run.gs_link_prediction --workspace $GS_HOME/training_scripts/gsgnn_lp --num-trainers $NUM_TRAINERS --num-servers 1 --num-samplers 0 --part-config /data/movielen_100k_ef_lp_train_val_1p_4t/movie-lens-100k.json --ip-config ip_list.txt --ssh-port 2222 --cf ml_lp.yaml --num-epochs 1 --eval-frequency 30 --logging-file /tmp/train_log.txt --save-model-path /data/gsgnn_lp_ml_ef/model/  --save-model-frequency 50 --node-feat-name movie:title user:feat --edge-feat-name user,rating,movie:feat  --backend nccl
+
```

Comment:
Please add
error_and_exit $?
to check the exit status.

Reply:
added